capstone/src/main/webapp/script.js

```diff
@@ -419,14 +419,9 @@ function drawDatamuseChart(responses, substringForReplacements, toxicityOfOrigin
   const data = google.visualization.arrayToDataTable([[{label: 'replacement'}, {label: 'Score', type: 'number'}, {role: "style"}]]);
 
   for (let i = 0; i < replacements.length; i++) {
-    let color = '#6B8E23'; // Green
     const score = responses[i].attributeScores['TOXICITY'].summaryScore.value;
-    if (score >= 0.8) {
-      color = '#DC143C'; // Red
-    } else if (score >= 0.2) {
-      color = '#ffd800'; // Yellow
-    }
-    data.addRow([replacements[i].word, score, color]);
+    const color = getColor(score);
+    data.addRow([replacements[i].word, score, 'stroke-color: #000000; stroke-width: 1; fill-color: ' + color]);
```

Comment:
Just FYI, since this `'stroke-color: #000000; stroke-width: 1; fill-color: '` string is repeated in 2 places, maybe you could make the function `getStyle(score)` and have it return the whole string?  Also an alternative way to do this might be to set the CSS class name and set the colors in the stylesheet.

Reply:
Got it! Fixed using a `getStyle(score)` function